manim/utils/space_ops.py

```diff
@@ -100,21 +131,67 @@ def quaternion_from_angle_axis(angle, axis, axis_normalized=False):
         return np.append(np.cos(angle / 2), np.sin(angle / 2) * normalize(axis))
 
 
-def angle_axis_from_quaternion(quaternion):
+def angle_axis_from_quaternion(quaternion: np.ndarray) -> float:
+    """Gets angle and axis from a quaternion.
+
+    Parameters
+    ----------
+    quaternion
+        The quaternion from which we get the angle and axis.
+
+    Returns
+    -------
+    float
+        Gives the angle and axis
+    """
     axis = normalize(quaternion[1:], fall_back=np.array([1, 0, 0]))
     angle = 2 * np.arccos(quaternion[0])
     if angle > TAU / 2:
         angle = TAU - angle
     return angle, axis
 
 
-def quaternion_conjugate(quaternion):
+def quaternion_conjugate(quaternion: np.ndarray) -> np.ndarray:
+    """Used for finding the conjugate of the quaternion
+
+    Parameters
+    ----------
+    quaternion
+        The quaternion for which you want to find the conjugate for.
+
+    Returns
+    -------
+    np.ndarray
+        Gives back the conjugate of the quaternion.
```

Comment:
```suggestion
        Returns the conjugate of the quaternion.
```

Reply:
Instead of having to returns twice we can just have "conjugate of the quaternion".

Reply:
LGTM